### PR TITLE
Add CLIP interactive tutorial (ndnly0915)

### PR DESCRIPTION
**Pull request by NDNLY**
## Summary

Adds a Simplified Chinese interactive tutorial for *Learning Transferable Visual Models From Natural Language Supervision* (CLIP), submitted as `learning_transferable_visual_models_from_natural_language_supervision/ndnly0915`.

The tutorial organizes the paper into ten chapters covering replaceable text candidates, the shared embedding space, contrastive pretraining, temperature scaling, prompt construction, zero-shot inference, the bidirectional objective, the dual-encoder architecture, transfer boundaries, and the reported distribution-shift results.

## Main interactions

- Switch the same kitten image between animal and action candidate sets.
- Drag normalized vectors to inspect cosine similarity.
- Explore the 3×3 image-text pairing matrix and both loss directions.
- Adjust the temperature and observe probabilities and cross-entropy.
- Step through zero-shot inference and dual-encoder projection stages.
- Compare the improved ResNet and Vision Transformer image branches.
- Inspect positive and negative transfer results and Figure 7 values.

All generated teaching images are stored locally and documented in the tutorial README. Teaching-only values are explicitly distinguished from measurements reported by the paper.

## Validation

- `npm run validate`
- `npm run catalog`, followed by restoration of `catalog/papers.json`
- `npm run validate:pr -- main`
- `npm run build:paper -- learning_transferable_visual_models_from_natural_language_supervision/ndnly0915`
- `git diff --check main...HEAD`
- Desktop and mobile preview, image loading, key interactions, and browser console checked; the participant confirmed the final preview.

## Preview

From the repository root, run:

```bash
npm run build:paper -- learning_transferable_visual_models_from_natural_language_supervision/ndnly0915
```

The expected public path after review, merge, and deployment is `papers/learning_transferable_visual_models_from_natural_language_supervision/ndnly0915/`.

### CODE REVIEW

**Comment by 1223BETTYYANG** [comment]
Merged.
